webapp/layouts/default.vue

```diff
@@ -3,13 +3,23 @@
     <div class="main-navigation">
       <ds-container class="main-navigation-container" style="padding: 10px 10px;">
         <div>
-          <ds-flex>
-            <ds-flex-item :width="{ base: '49px', md: '150px' }">
-              <nuxt-link to="/">
+          <ds-flex class="main-navigation-flex">
+            <ds-flex-item :width="{ lg: '5%' }" />
+            <ds-flex-item :width="{ base: '80%', sm: '80%', md: '80%', lg: '15%' }">
+              <a @click="$router.push('/').then(() => $router.go())">
                 <ds-logo />
-              </nuxt-link>
+              </a>
             </ds-flex-item>
-            <ds-flex-item>
+            <ds-flex-item
+              :width="{ base: '20%', sm: '20%', md: '20%', lg: '0%' }"
+              class="mobile-hamburger-menu"
+            >
+              <ds-button icon="bars" @click="toggleMobileMenuView" right />
```

Comment:
hope you are okay with me changing your implementation @ogerly... as I added the filter posts dropdown to the navbar, it cluttered and broke the css on mobile, so I just added a hamburger menu as it was always kinda needed.
iPhone:
![Screenshot at 2019-07-09 18:43:12](https://user-images.githubusercontent.com/26943915/60925264-8d7c8000-a279-11e9-94b1-54957a0f501b.png)
![Screenshot at 2019-07-09 18:44:00](https://user-images.githubusercontent.com/26943915/60925284-9ff6b980-a279-11e9-855b-35521697b85a.png)
![Screenshot at 2019-07-09 18:43:22](https://user-images.githubusercontent.com/26943915/60925285-a08f5000-a279-11e9-975e-9ffa025bfe8d.png)
Ipad:
![Screenshot at 2019-07-09 18:43:49](https://user-images.githubusercontent.com/26943915/60925299-abe27b80-a279-11e9-8d6a-34933f95dd5b.png)
![Screenshot at 2019-07-09 18:43:38](https://user-images.githubusercontent.com/26943915/60925300-abe27b80-a279-11e9-9ca1-9bc62b78d53a.png)